doc/text/news.md

```diff
@@ -1,5 +1,40 @@
 # NEWS
 
+## 6.0.0: 2016-03-06 {#version-6-0-0}
+
+### Improvements
+
+  * Supported Groonga 6.0.0. Groonga 5.1.2 or older aren't supported.
+  * Supported `-` operator as {Groonga::Operator::AND_NOT}.
+  * `grndump`: Added `:max_records` option.
+    [Patch by Genki Takiuchi]
+  * Accepted integer and time as function call arguments.
+  * `expression-builder`: Disabled auto ID detection.
+    It's confused when Int/Uint types for key.
```

Comment:
UInt notation is preferred, I think.
Because Groonga's `unsigned int` built-in series types are UInt[8|16|32|64] in reference: http://groonga.org/docs/reference/types.html